src/Analyzer.Reports/ConsoleReportWriter.cs

```diff
@@ -42,7 +42,7 @@ private void OutputToConsole(IEnumerable<Types.IEvaluation> evaluations)
                     string resultString = string.Concat(
                         GetFailedLines(evaluation)
                         .Select(l => $"{TwiceIndentedNewLine}Line: {l}"));
-                    var output = $"{IndentedNewLine}{(evaluation.RuleId != "" ? $"{evaluation.RuleId}: " : "")}{evaluation.RuleDescription}" +
+                    var output = $"{IndentedNewLine}{(evaluation.RuleId != "" ? $"{evaluation.RuleId}: " : "")}{evaluation.RuleShortDescription}" +
```

Comment:
Do you think we should put full too?

Reply:
mm no, I think it would be too much text on the console. This is how it looks now:

![image](https://user-images.githubusercontent.com/11780216/199846824-5023afe2-2a6a-4cf7-89f8-6ad9c3cea4c6.png)

Reply:
Some PSRule descriptions are pretty long, like 4 sentences